util/logtools/make_timeline.ts

```diff
@@ -22,10 +22,8 @@ import FFLogs, { ffLogsEventEntry } from './fflogs';
 
 // TODO: Add support for auto-generating loops.
 
-// TODO: Add FFLogs report parsing support.
-
-// TODO: Reinstate support for using raw start/end times
-// rather than force fight indexing.
+// TODO: We could consider re-adding `-s` and `-e` flags for manual start and end times,
+// but it would likely be better to just add support to `-lf` for any missing cases.
```

Comment:
Yes, I mostly left that in for edge cases. Sometimes when Ravahn changes stuff across expansions, it's nice not to have to figure out exactly how to update the collection logic if we just want a timeline Right Now. But that's probably not t hat much of a concern going forward.